README.md

```diff
@@ -55,8 +53,8 @@ docker run -p 80:80 -p 81:81 --volume /path/to/volume:/home/proxy --name cfbs-we
 
 ## CFEngine Build Repositories
 
-* [build-index](https://github.com/cfengine/build-index) - Index of modules
-* [build-website](https://github.com/cfengine/build-website) - Website
-* [cfbs](https://github.com/cfengine/cfbs) - Command line client
-* [modules](https://github.com/cfengine/modules) - Official modules provided by the CFEngine team
-* [module-template](https://github.com/cfengine/build-example) - Template for creating new modules
+- [build-index](https://github.com/cfengine/build-index) - Index of modules
+- [build-website](https://github.com/cfengine/build-website) - Website
+- [cfbs](https://github.com/cfengine/cfbs) - Command line client
+- [modules](https://github.com/cfengine/modules) - Official modules provided by the CFEngine team
+- [module-template](https://github.com/cfengine/build-example) - Template for creating new modules
```

Comment:
Please don't change this, it's copy-pasted in multiple repos, so easier to keep in sync if you don't.

Reply:
should now again be in sync with the rest 👌